Chatkit/Model/Message.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import CoreData
+
+public struct Message {
+    
+    public let identifier: String
```

Comment:
Everywhere else in the service (and pretty much in the world, as far as I can tell), this is abbreviated at `id` :)

Reply:
`id` is a reserved keyword in...Objective-C therefore it is not widely used on the platform.

Reply:
Oh, nice...